src/utils/denomination.ts

```diff
@@ -11,40 +11,47 @@ export const getValidationRegex = (maximum: number): RegExp => {
     const digits = maximum.toString().length;
     const firstDigit = BigNumber(maximum).div(satFactor).toString().charAt(0);
     const firstDigitRegex = firstDigit === "0" ? `0` : `0-${firstDigit}`;
-    const regex = `^[0-9]{1,${digits}}$|^[${firstDigitRegex}](\\.[0-9]{1,8}){0,1}$`;
+    const regex = `^[0-9]{1,${digits}}$|^[${firstDigitRegex}]((\\.|,)[0-9]{1,8}){0,1}$`;
     return new RegExp(regex);
 };
 
 export const formatAmount = (
     amount: BigNumber,
     denomination: string,
+    separator: string = ".",
```

Comment:
Not a fan of having a default. Could lead to us missing setting the parameter in some cases